cmd/dbbench/main.go

```diff
@@ -164,8 +180,7 @@ func main() {
 		*threads = *iter
 	}
 
-	// Use built-in benchmarks.
```

Comment:
This was causing database benchmarkers that did not have default benchmarks to fail (like mssql).